docs/refs.bib

```diff
@@ -48,6 +106,49 @@ @inproceedings{hardt2016equality
   url       = {https://proceedings.neurips.cc/paper/2016/hash/9d2682367c3935defcb1f9e247a97c0d-Abstract.html}
 }
 
+
+@inproceedings{madaio2020codesigning,
+  title={Co-designing checklists to understand organizational challenges and opportunities around fairness in AI},
+  author={Madaio, Michael A and Stark, Luke and Wortman Vaughan, Jennifer and Wallach, Hanna},
+  booktitle={{ACM CHI}},
+  pages={1--14},
+  year={2020},
+  url={https://dl.acm.org/doi/abs/10.1145/3313831.3376445}
+}
+
+
+@article{mehrabi2021survey,
+  title={A survey on bias and fairness in machine learning},
+  author={Mehrabi, Ninareh and Morstatter, Fred and Saxena, Nripsuta and Lerman, Kristina and Galstyan, Aram},
+  journal={ACM Computing Surveys (CSUR)},
+  volume={54},
+  number={6},
+  pages={1--35},
+  year={2021},
+  publisher={ACM New York, NY, USA},
+  url={https://arxiv.org/abs/1908.09635}
+}
+
+
+@incollection{noble2018algorithms,
+  title={Algorithms of oppression},
+  author={Noble, Safiya Umoja},
+  booktitle={Algorithms of oppression},
+  year={2018},
+  publisher={New York University Press},
+  url={https://nyupress.org/9781479837243/algorithms-of-oppression/}
+}
+
+
+@book{o2017weapons,
+  title={Weapons of math destruction: How big data increases inequality and threatens democracy},
+  author={O'neil, Cathy},
```

Comment:
```suggestion
  author={O'Neil, Cathy},
```

Reply:
or link to the publisher, as we do for some other books:
https://www.penguinrandomhouse.com/books/241363/weapons-of-math-destruction-by-cathy-oneil/